charts/rancher-monitoring/v0.1.3/charts/webhook-receiver/templates/deployment.yaml

```
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+{{- include "webhook-receiver.labels" . | nindent 4 }}
```

nindent should create a new line.

```suggestion
  labels: {{- include "webhook-receiver.labels" . | nindent 4 }}
```